latex/Inhalt/analyse.tex

```diff
@@ -1 +1,30 @@
 \chapter{Analyse}
+
+\section{Zielstellung}\label{goals}
+
+Das Ziel ist es einen Emulator zu entwickeln, welcher die vollständige Intel 8080 Spezifikation unterstützt. Dabei sind die zentralen Aspekte wie folgt:
+
+\begin{itemize}
+    \item Vollständige 8080 Assembly Unterstützung
+    \item Simulierte Schnittstelle zu Ein-/Ausgabegeräten
+    \item Korrekte Behandlung von Hardware-Interrupts
+\end{itemize}
+
+
+
+Außerdem soll ein entsprechendes Web-Frontend entwickelt werden, um den Emulator zu bedienen.
+Dieses soll einen Editor beinhalten, um Assembly Programme zu schreiben, die Ausführung dieser Programme ermöglichen, den Zustand des Emulators während der Ausführung darstellen und Auswahl zwischen verschiedenen Peripherie-Geräten ermöglichen (Pixel-Display, Eingabefeld, o.ä.). 
+
+Es soll sowohl möglich sein Schritt für Schritt durch ein Programm zu gehen, als auch das Programm automatisch laufen zu lassen.
+
+\section{Beitrag}
+
+Unser Beitrag ist \Emu, ein in Rust geschriebener Intel 8080 Emulator mit Web-Frontend. \Emu erfüllt die gestellten Ziele vollständig.
+
+Dadurch, dass unser Emulator nach Web-Assembly kompiliert wird, läuft der Emulator naiv im Browser des Clienten. Durch eine explizit definierte Schnitstelle, kann der Emulator aus dem JavaScript Code bedient werden.
+
+Der Editor unterstützt Syntax-Highlighting und Code-Completion und verügt über Buttons um Kompilation und Ausführung des Programmes zu ermöglichen.
+Es gibt Anzeigen für den Zustand der CPU (Register, Flaggen, etc.), für den Arbeitsspeicher und für die Peripherie-Geräte.\\
+
+In \cref{chap:design} wird unser Design für den Emulator, die API und das Frontend
+erläutert. Darauf folgend werden in \cref{chap:impl}
```

Comment:
The cliffhander is due to the emulator not being quite done, I stopped writing this chapter because i realized much of what I'm writing is still subject to change.